### PR TITLE
Using the Device Size in Conditions

### PR DESCRIPTION
**Pull request by MapleSyyrup**
- Add other design depending on the width of the device regardless which
  device orientation is used. If the device has more than 460 pixels width, 'Delete' label beside the delete button is
  seen. Otherwise, only the delete button can be seen.

image

In landscape mode, the width is more than 460 that's why the Delete word can be seen.
image

### CODE REVIEW

**Comment by Nei1eveN** [comment]
> * Add other design depending on the width of the device regardless which
>   device orientation is used. If the device has more than 460 pixels width, 'Delete' label beside the delete button is
>   seen. Otherwise, only the delete button can be seen.
> 
> image
> 
> In landscape mode, the width is more than 460 that's why the Delete word can be seen.
> image

@MapleSyyrup does the `show chart` here toggle works now?

File: lib/widgets/transaction_list.dart [hunk 1]
```diff
@@ -28,37 +28,43 @@ class TransactionList extends StatelessWidget {
...
-                        padding: EdgeInsets.all(6),
-                        child: FittedBox(
-                          child: Text('\$${transactions[index].amount}'),
-                        ),
+          })
+        : ListView.builder(
+            itemBuilder: (ctx, index) {
+              return Card(
+                elevation: 5,
+                margin: EdgeInsets.symmetric(vertical: 6, horizontal: 7),
```
Comment (Nei1eveN) [inline lib/widgets/transaction_list.dart]: trailing commas

File: lib/widgets/transaction_list.dart [hunk 2]
```diff
@@ -28,37 +28,43 @@ class TransactionList extends StatelessWidget {
...
+                child: ListTile(
+                  leading: CircleAvatar(
+                    radius: 30,
+                    child: Padding(
+                      padding: EdgeInsets.all(6),
```
Comment (Nei1eveN) [inline lib/widgets/transaction_list.dart]: `const`

**Review by Nei1eveN — COMMENTED** [review]
@MapleSyyrup Please check my comments

**Comment by MapleSyyrup** [comment]
> > * Add other design depending on the width of the device regardless which
> >   device orientation is used. If the device has more than 460 pixels width, 'Delete' label beside the delete button is
> >   seen. Otherwise, only the delete button can be seen.
> > 
> > image
> > In landscape mode, the width is more than 460 that's why the Delete word can be seen.
> > image
> 
> @MapleSyyrup does the `show chart` here toggle works now?

@Nei1eveN yes but when the chart is shown, the listed transactions can't be seen. This will be fixed on next update.

File: lib/widgets/transaction_list.dart (hunk 1 above)
Comment (Nei1eveN) [inline lib/widgets/transaction_list.dart]: `const`?

**Review by Nei1eveN — APPROVED** [review]

**Pull request merged**